lib/order.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'menu'
+
+class Order
+  def add_dish(menu, dish, quantity)
+    if menu.dish_list.include?(dish)
+    "You have ordered #{quantity} #{dish}"
+  else
+    raise "Error you must select an item on the menu"
```

Comment:
Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)